integrity-backend/input-browsertrix-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,21 @@
+{
+  "contentMetadata":
+  {
+    "name": "Web archive of @rt_com Twitter",
+    "mime": "application/wacz",
+    "description": "Web archive of @rt_com Twitter on 2022-03-29",
+    "createdBy": "Starling Lab",
+    "created": "2022-03-29T13:54:20Z",
+    "extras":
+    {
+      "authsignSoftware": "authsigner 0.3.1",
+      "authsignDomain": "authsign.stg.starlinglab.org",
+      "waczVersion": "1.1.1",
+      "pages":
+      {
+        "5ZwqT9xGkYVdYku3EVQLrD": "https://twitter.com/rt_com"
```

Comment:
what is `5ZwqT9xGkYVdYku3EVQLrD`

Reply:
Ah yes

this data came from the `pages/pages.jsonl` file inside the warcz file

However I recently found out that the `pages/pages.jsonl` file is missing win the warcz sometimes <- ping @ikreymer 

Had to check do a check if its there. So i dont know if you want to use it or not.
```

                if "pages/pages.jsonl" in wacz.namelist():
                    with wacz.open("pages/pages.jsonl") as jsonl_file:
                        for line in jsonl_file.readlines():
                            d = json.loads(line)
                            if "url" in d:
                                content_meta["pages"][d["id"]] = d["url"]
```

Source: https://github.com/starlinglab/integrity-preprocessor/blob/dev/browsertrix/main.py#L366-L371

Reply:
Don't mind using it. Also okay with changing format to something like:

```
"pages": [ "https://twitter.com/rt_com" ]
```

If you think the hash is not useful. This thing ends up on registration chains so idk.